### PR TITLE
custom fields portal

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
fix portal fields

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/brand/src/entities/RoutingPattern/RoutingPattern.tsx [hunk 1]
```diff
@@ -5,24 +5,25 @@ import defaultEntityBehavior from '@irontec/ivoz-ui/entities/DefaultEntityBehavi
...
+  RoutingPatternPropertyList,
+} from './RoutingPatternProperties';
+import { getI18n } from 'react-i18next';
 import foreignKeyResolver from './ForeignKeyResolver';
+import { EntityValues } from '@irontec/ivoz-ui';
 
 const properties: RoutingPatternProperties = {
   prefix: {
     label: _('Prefix'),
-    pattern: new RegExp('/+[0-9]*/'),
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/RoutingPattern/RoutingPattern.tsx:14]: why?

File: web/portal/brand/src/translations/ca.json [hunk 2]
```diff
@@ -57,6 +57,8 @@
   "Brand": "Marca",
   "Brand service_one": "Servei de marca",
   "Brand service_other": "Serveis de marca",
+  "Brute-force attacks_one": "",
```
Comment (mmadariaga) [inline web/portal/brand/src/translations/ca.json:60]: set translations

File: web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx [hunk 3]
```diff
@@ -5,16 +5,9 @@ import {
 } from '@irontec/ivoz-ui/services/entity/EntityService';
 
 export type ActiveCallsPropertyList<T> = {
-  Event?: T;
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx:8]: why?

File: web/portal/platform/src/entities/Brand/ForeignKeyGetter.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      'features',
+      'proxyTrunks',
+      'defaultTimazone',
+      'language',
+      'currency',
+    ],
+  });
+
+  promises[promises.length] = featureSelectOptions({
+    callback: (options: any) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/ForeignKeyGetter.tsx:30]: any is forbidden

File: web/portal/platform/src/entities/Currency/ForeignKeyGetter.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { CurrencyPropertyList } from './CurrencyProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
+export const foreignKeyGetter: ForeignKeyGetterType = async ({
+  cancelToken,
+  entityService,
+}): Promise<any> => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Currency/ForeignKeyGetter.tsx:9]: any is forbidden

File: web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { SpecialNumberPropertyList } from './SpecialNumberProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx:5]: remove this file

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx (hunk 3 above)
Comment (fmendozzac) [reply web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx:8]: image

File: web/portal/brand/src/entities/RoutingPattern/RoutingPattern.tsx (hunk 1 above)
Comment (fmendozzac) [reply web/portal/brand/src/entities/RoutingPattern/RoutingPattern.tsx:14]: it does not work

File: web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx (hunk 6 above)
Comment (fmendozzac) [reply web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx:5]: it is needed for the form

File: web/portal/brand/src/entities/RoutingPattern/RoutingPattern.tsx (hunk 1 above)
Comment (mmadariaga) [reply web/portal/brand/src/entities/RoutingPattern/RoutingPattern.tsx:14]: fix it then

File: web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx (hunk 3 above)
Comment (mmadariaga) [reply web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx:8]: this is another entity actually, data come from a web socket service, not the API (https://github.com/irontec/ivozprovider/blob/bleeding/web/portal/platform/src/router/addCustomRoutes.ts) . Will handle it on it's own issue. Revert this for now

File: web/portal/platform/src/entities/Currency/ForeignKeyGetter.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { CurrencyPropertyList } from './CurrencyProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Currency/ForeignKeyGetter.tsx:5]: remove this file

File: web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx (hunk 6 above)
Comment (mmadariaga) [reply web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx:5]: use the default one 
`import { foreignKeyGetter } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';`

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/brand/src/translations/en.json [hunk 8]
```diff
@@ -57,6 +57,8 @@
   "Brand": "Brand",
   "Brand service_one": "Brand service",
   "Brand service_other": "Brand services",
+  "Brute-force attacks_one": "Brute-force attacks_one",
+  "Brute-force attacks_other": "Brute-force attacks_other",
```
Comment (mmadariaga) [inline web/portal/brand/src/translations/en.json:61]:
```suggestion
  "Brute-force attacks_one": "Brute-force attacks",
  "Brute-force attacks_other": "Brute-force attacks",
```

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**